app/controllers/interview_surveys_controller.rb

```diff
@@ -0,0 +1,10 @@
+class InterviewSurveysController < ApplicationController
+  def show
+    if Feature.enabled?(:survey_feature, current_user)
+      # code to show the survey if not hardcoded into the view
+    else
+      # this does not work
+      redirect_to home_path, notice: 'Feature not enabled'
```

Comment:
Just wanna make sure I'm understanding the comment `# this does not work`. Are we indicating that the logic won't work for the user that tries to get to this and doesn't have the feature enabled?

Or is the comment indicating that the redirect isn't actually working as is?

Reply:
The redirect isn't working as is. If you disable the feature and reload the page, it still loads the page and does not redirect.

I could remove this conditional code from the controller, at least for the MVP, as we can use the feature flag check in the view.

Reply:
I went ahead and removed the conditional code from the controller for now.